cmd/zk_quorum_check.go

```diff
@@ -0,0 +1,132 @@
+// Copyright © 2017 Mesosphere Inc. <www.mesosphere.com>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/dcos/dcos-checks/client"
+	"github.com/dcos/dcos-go/dcos"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// zookeeperQuorumCmd represents the zookeeper-quorum command
+var zookeeperQuorumCmd = &cobra.Command{
+	Use:   "zk-quorum",
+	Short: "DC/OS check zookeeper in quorum",
+	Long:  `Zookeeper quorum`,
+	Run: func(cmd *cobra.Command, args []string) {
+		RunCheck(NewZookeeperQuorumCheck("DC/OS zookeeper quorum check"))
+	},
+}
+
+func init() {
+	RootCmd.AddCommand(zookeeperQuorumCmd)
+}
+
+// ZkResponse struct response for localhost:8181/exhibitor/v1/cluster/state/<host>
+type ZkResponse struct {
+	Response struct {
+		Switches struct {
+			Restarts bool `json:"restarts"`
+			Cleanup  bool `json:"cleanup"`
+			Backups  bool `json:"backups"`
+		} `json:"switches"`
+		State       int    `json:"state"`
+		Description string `json:"description"`
+		IsLeader    bool   `json:"isLeader"`
+	} `json:"response"`
+	ErrorMessage string `json:"errorMessage"`
+	Success      bool   `json:"success"`
+}
+
+// ZookeeperQuorumCheck struct
+type ZookeeperQuorumCheck struct {
+	Name string
+}
+
+// Run the check
+func (zk *ZookeeperQuorumCheck) Run(ctx context.Context, cfg *CLIConfigFlags) (string, int, error) {
```

Comment:
I'd recommend naming the return args here so that it's clear what the values represent.